src/drt/src/dr/FlexGridGraph.h

```diff
@@ -1009,6 +1011,7 @@ class FlexGridGraph
   frVector<frLayerNum> zCoords_;
   frVector<frCoord> zHeights_;  // accumulated Z diff
   std::vector<dbTechLayerDir> layerRouteDirections_;
+  std::vector<bool> layerRouteDirectionsHorizontal_;
```

Comment:
vector<bool> does bit packing which I suspect would make it slower than vector<char>.  Would you try that out and see?